scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -149,6 +149,11 @@ public static BuildTargetResult RestoreLockedCoreHost(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target(
+            nameof(CompileCoreHost),
+            nameof(BuildCoreHostPackages))]
+        public static BuildTargetResult BuildCoreHost(BuildTargetContext c) => c.Success();
```

Comment:
They are still 2 independent targets.  You can call CompileCoreHost and then call BuildCoreHostPackages.

This just simplifies the Compile target to not have to know about 5 different "host" targets.